### PR TITLE
Print error in buildscript to stderr

### PR DESCRIPTION
**Pull request by olehermanse**
Signed-off-by: Ole Herman Schumacher Elgesem <ole.elgesem@cfengine.com>

### CODE REVIEW

**Comment by olehermanse** [comment]
@jimis please review

File: deps-packaging/pkg-cache [hunk 1]
```diff
@@ -34,7 +34,7 @@ firstlabel=${NODE_LABELS%% *}
 
 if [ x"$firstlabel" = x ]
 then
-    echo "Jenkins has not assigned a LABEL to this node, you are probably building manually"
+    (>&2 echo "Jenkins has not assigned a LABEL to this node, you are probably building manually")
```
Comment (jimis) [inline deps-packaging/pkg-cache]: Please don't use bash-isms. ;-)

**Comment by olehermanse** [comment]
@jimis happy?

**Comment by jimis** [comment]
> @jimis happy?

In general no, but this PR is up to standards now so it's merged. ;-)

**Pull request merged**